model/modules/path_classifier.py

```diff
@@ -23,7 +23,9 @@ def __init__(self, config: ClassifierConfig, out_size: int):
         self.out_size = out_size
         self.attention = LocalAttention(config.classifier_input_size)
         layers = [self._get_activation(config.activation), nn.Linear(config.classifier_input_size, config.hidden_size)]
-        for _ in range(config.n_hidden_layers):
+        if config.n_hidden_layers < 1:
+            raise ValueError(f"Invalid layers number ({config.n_hidden_layers})")
+        for _ in range(config.n_hidden_layers - 1):
             layers += [self.activations[config.activation], nn.Linear(config.hidden_size, config.hidden_size)]
```

Comment:
Use here `_get_activation` function

Reply:
Also, the linear layer should be before activation.
So, the sequence is attention, linear, activation, ..., linear, activation, projection.